x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_some_domains_modal/simplified_selectable.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiSelectable } from '@elastic/eui';
+import { EuiSelectableLIOption } from '@elastic/eui/src/components/selectable/selectable_option';
+
+export interface Props {
+  options: string[];
+  selectedOptions: string[];
+  onChange(selectedOptions: string[]): void;
+}
+
+export interface OptionMap {
+  [key: string]: boolean;
+}
+
+export const SimplifiedSelectable: React.FC<Props> = ({ options, selectedOptions, onChange }) => {
```

Comment:
We'll be re-using this component for the flyout we'll be implementing in the next phase.